ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/baseActivityPicklistQuestion.component.ts

```diff
@@ -1,9 +1,12 @@
-import { Input, Output, EventEmitter, OnChanges, SimpleChanges, OnDestroy } from '@angular/core';
+import { Input, Output, EventEmitter, OnChanges, SimpleChanges, OnDestroy, Component } from '@angular/core';
 import { ActivityPicklistQuestionBlock } from '../../../models/activity/activityPicklistQuestionBlock';
 import { ActivityPicklistAnswerDto } from '../../../models/activity/activityPicklistAnswerDto';
 import { NGXTranslateService } from '../../../services/internationalization/ngxTranslate.service';
 import { Subscription } from 'rxjs';
 
+@Component({
+    template: ``
+})
```

Comment:
Angular 10 requires decorator if we want to use component features e.g. `@Input()`